### DISCUSSION TITLE
AzerothCore latest (Player Bots)

### DISCUSSION THREAD

**Discussion by speedrunz** [azerothcore/azerothcore-wotlk#3945] [Job offers]
Hello,

I hope that you are having a great day.

I am looking for a PlayerBots / fake players online patch that works on the latest AzerothCore.

Something like this -> https://www.youtube.com/watch?v=sbRdxWtVf5M . If you have something better, much better.

- They should be whisperable.
- Changes location after 6 minutes
- They should move like normal players, like they're questing/levelling.

Budget/Price starts at $150, will increase it with no problem.

**Comment by Kitzunu** [comment]
https://github.com/azerothcore/azerothcore-wotlk/issues/2159
